jenkins-pipelines/perf-regression-throughput-shard-aware-30min-gce.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "gce",
+    region: "us-east1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression-gcp.100threads.30M-keys.yaml", "configurations/run_db_node_benchmarks.yaml", "configurations/disable_speculative_retry.yaml"]''',
+    sub_tests: ["test_write", "test_read", "test_mixed"],
+    test_email_title: "throughput - non-shard-aware",
+
+    timeout: [time: 350, unit: "MINUTES"]
```

Comment:
Timeout i think is not needed any more, and may be it is good to have in title `GCP`

Reply:
SCT_GCE_IMAGE_DB=image SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo' ./sct.py lint-yamls -b **gce** -i 'rolling,artifacts,private-repo,**gce**,jepsen' -e 'multi-dc,multiDC,docker,azure'